modules/apps/commerce/commerce-order-web/src/main/java/com/liferay/commerce/order/web/internal/display/context/CommerceOrderEditDisplayContext.java

```diff
@@ -463,6 +463,21 @@ public BigDecimal getDecimalQuantity(CommerceOrderItem commerceOrderItem) {
 			_commerceOrderItemDecimalQuantityConfiguration.roundingMode());
 	}
 
+	public String getDescriptiveAccountInfo(CommerceAccount commerceAccount) {
+		if (commerceAccount == null) {
+			return StringPool.BLANK;
+		}
+
+		StringBundler sb = new StringBundler(4);
+
+		sb.append(commerceAccount.getName());
+		sb.append(StringPool.COMMA_AND_SPACE);
```

Comment:
Looking at the wireframes I see the ID in a new line under the account name.
I guess you should put a
sb.append(StringPool.NEW_LINE);
if you want to use this code.